### DISCUSSION TITLE
Failure to export from collector to backend (GO Lambda Layer)

### DISCUSSION THREAD

**Discussion by itaybaror** [open-telemetry/opentelemetry-collector#5741] [Q&A]
I have been attempting to instrument lambdas using the docs on the ADOT website. I have successfully instrumented some python lambdas, where they send the data to the collector, and then exports the data to lightstep (my backend). However when i try to instrument in go, i do see the logs in cloudwatch (i configured the collector to send data to lightstep and to logs) but when it tries to export the data to lightstep (and i have tried other backends) i am usually greeted with some sort of grpc or timeout error, and i do not see the data in the backend. 

I have attached pictures with the errors i receive. Keep in mind that with my working python implementation i am using the same collector config file.

Errors:
<img width="1426" alt="Screen Shot 2022-07-25 at 10 04 16 AM" src="https://user-images.githubusercontent.com/107562167/180797738-5e691af1-5139-40bb-a51b-e9b674b2df10.png">

<img width="1313" alt="Screen Shot 2022-07-25 at 10 12 26 AM" src="https://user-images.githubusercontent.com/107562167/180797956-6ff8b557-ac0a-4b16-82f2-28730b6a7652.png">

My collector config file:
<img width="1051" alt="Screen Shot 2022-07-25 at 10 11 41 AM" src="https://user-images.githubusercontent.com/107562167/180797823-88927d34-3129-4bfd-9d06-9c3cbf2bf4e8.png">

**Comment by arabzade** [comment]
Any update here? I am having the similar issue
